courses/templates/courses/course_comparison_page.html

```diff
@@ -0,0 +1,103 @@
+{% extends "base.html" %}
+{% load staticfiles sass_tags wagtailcore_tags discover_uni_tags %}
+
+
+{% block extra_css %}
+    <link href="{% sass_src 'scss/course_comparison.scss' %}" rel="stylesheet" type="text/css" >
+    <link href="{% sass_src 'scss/course_detail.scss' %}" rel="stylesheet" type="text/css" >
+{% endblock %}
+
+{% block extra_js %}
+    <script type="text/javascript" src="{% static 'js/course_comparison.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/course_detail_accordions.js' %}"></script>
+    <script src="https://code.highcharts.com/highcharts.js"></script>
+    <script src="https://code.highcharts.com/highcharts-more.js"></script>
+    <script src="https://code.highcharts.com/modules/solid-gauge.js"></script>
+    <script type="text/javascript" src="{% static 'js/doughnut_chart.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/bar_chart.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/label_explanation_popups.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/overview_blocks.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+    <div class="comparison discover-uni-container">
+        <div class="comparison__overview">
+            <h1 class="comparison__course-count">
+                <span class="count"></span> {{page.heading}}
+            </h1>
+
+            <p class="comparison__lead">
+                {{page.lead}}
+            </p>
+        </div>
+
+        <div class="comparison__nav-box">
+            {{page.remove_text | richtext}}
+
+            <!--{{page.save_text | richtext}}-->
```

Comment:
Should this be commented out?

Reply:
Spoken to Matt - used to share course comparison - functionality does not yet exist hence line commented out